tutorials/labs/gpu-partitioning.md

```
@@ -92,10 +94,48 @@ spec:
+kubectl apply -f /home/ubuntu/gpumem-pod-a.yaml -f /home/ubuntu/gpumem-pod-b.yaml
+kubectl get pods gpumem-pod-a gpumem-pod-b -o wide
+```
+
+</TabItem>
+<TabItem value="gcp" label="GCP">
+
 ```bash
 kubectl apply -f tutorials/labs/examples/03-gpu-partitioning/gpumem-pod-a.yaml -f tutorials/labs/examples/03-gpu-partitioning/gpumem-pod-b.yaml
+
 kubectl get pods gpumem-pod-a gpumem-pod-b -o wide
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)